documentation/docs/Exercises/_category_.json

```diff
@@ -1,4 +1,4 @@
 {
-  "label": "Exercises",
+  "label": "Exercises (coming soon)",
   "position": 4
-}
+}
```

Comment:
Missing newline

Reply:
My IDE keeps stripping away the newline.  I'll add it via this github comment 😛 
```suggestion
}

```